src/Pages/SearchResult/SearchResult.js

```diff
@@ -1,120 +1,105 @@
 import React, { Component } from "react";
+import ResultFrame from "./ResultFrame";
 import PremiumZone from "./PremiumZone";
 import PlusZone from "./PlusZone";
 import NormalZone from "./NormalZone";
+import Nav from "../../Components/Nav";
 import "./SearchResult.scss";
 
-const OPTIONS = [
-  { id: "1", type: "공간 유형", option: "모든 공간", icon: "downArrow" },
-  { id: "2", type: "지역", option: "전체", icon: "downArrow" },
-  { id: "3", type: "이용일", option: "모든 날짜", icon: "calendarIcon" },
-];
-
-const CATEGORY = [
-  { list: "전체", bar: true },
-  { list: "시간단위", bar: true },
-  { list: "패키지단위", bar: true },
-  { list: "월단위", bar: false },
-];
-
-const SELECT = [
-  { value: "first", order: "베스트 공간 순" },
-  { value: "second", order: "가격 낮은 순" },
-  { value: "third", order: "가격 높은 순" },
-  { value: "fourth", order: "이용후기 많은 순" },
-];
-
-const BUTTON = [
-  { id: "1", icon: "locationIcon", name: "지도" },
-  { id: "2", icon: "filterIcon", name: "필터" },
-];
+const LIMIT = 6;
 
 class SearchResult extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
     this.state = {
+      premium: [],
+      plus: [],
+      normal: [],
       data: [],
+      queryString: decodeURI(props.location.search.split("=")[1]),
+      offset: 0,
     };
   }
+
   componentDidMount() {
-    fetch("http://localhost:3000/data/data.json")
-      .then((res) => res.json)
+    this.getSearchData();
+    window.addEventListener("scroll", this.getData);
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (prevProps.location.search !== this.props.location.search) {
+      this.setState(
+        {
+          queryString: decodeURI(this.props.location.search.split("=")[1]),
+        },
+        () => this.getSearchData()
+      );
+    }
+  }
+
+  getSearchData = () => {
+    const { queryString, offset } = this.state;
+    fetch(`http://192.168.219.106:8001/spaces/premium?search=${queryString}`)
```

Comment:
이 주소도 계속 바껴서 config 파일 안에서 변수로 넣고 관리해주세요!
```suggestion
    fetch(`${API}/spaces/premium?search=${queryString}`)
```

Reply:
네 수정하였습니다!